docs-markdown/snippets/snippets.json

```diff
@@ -1,12 +1,37 @@
 {
   "rows": {
     "prefix": "row",
-    "body": [":::row:::\n\n:::row-end:::"],
+    "body": [
+      ":::row:::\n\n:::row-end:::"
+    ],
     "description": "row"
   },
   "columns": {
     "prefix": "column",
-    "body": ["    :::column:::\n\n    :::column-end:::"],
+    "body": [
+      "    :::column:::\n\n    :::column-end:::"
+    ],
     "description": "column"
+  },
+  "content": {
+    "prefix": "image",
+    "body": [
+      ":::image source=\"\" alt-text=\"\":::"
```

Comment:
even though type="content" is supposed to be the default. The actual json schema requires the type property. I think we should add it into the snippet with type="content"

Reply:
Sounds good @bharney0 , updated.